src/archBuild_sconscript

```diff
@@ -33,11 +33,21 @@ sources = [
 ]
 
 if env["PLATFORM"] == "win32":
-	env.Append(CXXFLAGS="/EHsc /std:c++20 /W3 /WX "
-		# #479: fmt/format.h includes UTF-8 characters in comments. However, cl
-		# treats the file as ANSI unless it has a BOM. This causes a warning (and
-		# thus an error) on systems with certain locales. Just disable this warning.
-		"/wd4819")
+	# On Windows, OSARA is build with LLVM to have a toolchain that's closer to what's used on Mac
+	env["CC"] = "clang-cl"
+	env["LINK"] = "lld-link"
+	env.Append(CXXFLAGS=[
+		"/clang:-std=c++20",
+		"/EHsc",
+		"/W3",
+		"/WX",
+		# Ignore some warnings (treated as errors)
+		# Clang warns about unused functions in WDL/wdltypes.h
+		"/clang:-Wno-unused-function",
+		# Ignore suggestions to add braces around initialization of subobjects. We might want to fix this at some point
+		"/clang:-Wno-missing-braces",
+	])
+	env.Append(CPPDEFINES=["_CRT_SECURE_NO_WARNINGS", "_CRT_NONSTDC_NO_WARNINGS"])
```

Comment:
Hmm. Why didn't we need this with MSVC? Any idea?

Reply:
I've looked into it but I have no idea. I wonder whether msvc automatically choses the secure equivalents, in which case switching to Clang would make this less secure.